python/tests/voice/test_elevenlabs_pump.py

```
@@ -0,0 +1,469 @@
+# --------------------------------------------------------------------------- #
+# A8 — task cleanup: no orphan                                                #
+# --------------------------------------------------------------------------- #
+
+
+@pytest.mark.asyncio
+async def test_pump_task_cancelled_and_awaited_on_disconnect() -> None:
+    """A8: after disconnect the pump task is cancelled AND awaited (done), and
+    no pending-task warning is emitted."""
+    with warnings.catch_warnings(record=True) as caught:
+        warnings.simplefilter("always")
+        adapter, socket = await _connected_adapter()
```

Ruff (0.15.20)

[warning] Unpacked variable `socket` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)